### DISCUSSION TITLE
Curious about --no-dll option

### DISCUSSION THREAD

**Discussion by speelbarrow** [storybookjs/storybook#12898] [Help] [answered]
After updating to React 17, it seems like Storybook will only run correctly when supplied with the `--no-dll` argument (see #12408). I'm curious what the `no-dll` option actually does and why it fixes this issue.

**Comment by tooppaaa** [comment, marked as answer]
It doesn't load the pre-compiled sources of storybook's UI and docs.
Part of https://github.com/storybookjs/storybook/pull/12874 was to disable the docs dll by default and let you compile everything.

This was meant for performance optimization but a different road is under investigation.
More infos on dll: https://webpack.js.org/plugins/dll-plugin/
